content/docs/1.5.0/spdk/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SPDK Data Engine (Preview Feature)
+weight: 3
+---
+
+## Platform Support
+
+Currently, SPDK Data Engine only supports `x86_64` platform.
+
+## Feature Support
+
+- Volume basic lifecycle (creation, attachment, detachment and deletion)
+- Offline replica rebuilding
```

Comment:
- Volume lifecycle (creation, attachment, detachment and deletion)
- Degraded volume
- Offline replica rebuilding
- Block disk management

Reply:
Updated. Thank you.